components/dashboard/src/admin/WorkspacesSearch.tsx

```diff
@@ -104,7 +96,9 @@ export function WorkspaceSearch(props: Props) {
                             <path fillRule="evenodd" clipRule="evenodd" d="M6 2a4 4 0 100 8 4 4 0 000-8zM0 6a6 6 0 1110.89 3.477l4.817 4.816a1 1 0 01-1.414 1.414l-4.816-4.816A6 6 0 010 6z" fill="#A8A29E" />
                         </svg>
                     </div>
-                    <input type="search" placeholder="Search Workspaces" onKeyDown={(ke) => ke.key === 'Enter' && search() } onChange={(v) => { setQueryTerm(v.target.value) }} />
+                    <input type="search" placeholder="Search Workspace IDs or Instance IDs" style={{width: "20rem"}}
```

Comment:
**question**: Shall we go with one of these here based on https://github.com/gitpod-io/gitpod/issues/8453#issuecomment-1055427520? Cc @geropl